telematic_system/historical_data_processing/rosbag2_processing_service/pytest.ini

```diff
@@ -0,0 +1,17 @@
+; Environment variables defined for unit tests
+[pytest]
+TOPIC_EXCLUSION_LIST=["/rosout","/parameter_events","/robot_description","/tf_static","/tf","/environment/fused_external_objects","/environment/base_map","/environment/external_objects_viz","/environment/motion_computation_visualize","/environment/external_object_predictions","/localization/current_pose","/localization/map_param_loader/georeference","/environment/semantic_map","/hardware_interface/comms/outbound_binary_msg","/environment/lanelet2_map_viz","/localization/points_map"]
```

Comment:
Is this file needed?

Reply:
Yes i'm setting environment variables to use for unit tests here